addons/mail/static/src/components/message/message_tests.js

```diff
@@ -144,11 +145,14 @@ QUnit.test('moderation: as author, moderated channel with pending moderation mes
 QUnit.test('moderation: as moderator, moderated channel with pending moderation message', async function (assert) {
     assert.expect(9);
 
+    this.data['mail.channel'].records.push({
+        id: 20,
+        is_moderator: true,
+    });
     await this.start();
-    const thread = this.env.models['mail.thread'].create({
+    const thread = this.env.models['mail.thread'].findFromIdentifyingData({
         id: 20,
         model: 'mail.channel',
-        moderators: [['link', this.env.messaging.currentPartner]],
     });
     const message = this.env.models['mail.message'].create({
```

Comment:
Here an example where another models create is done in tests, that should ideally be removed too.

But it's fine for me if we do it progressively, I'd rather review once multiple small PR, than many times a big one.

We can spot some of them with search `.create(` in `*tests.js`.